pyproject.toml

```diff
@@ -40,7 +40,7 @@ dependencies = [
   "shapely",
   "spatialpandas",
   "geopandas",
-  "xarray",
+  "xarray>=2024.11.0",
```

Comment:
Why is this needed for?

Reply:
For the `ufunc` module. I'm not sure if there's an older method we could use that wouldn't require the pin.

Reply:
Or, would that be helpful for future to put a comment into the file about this?

Reply:
That's a good idea. I'm going to check if there's a better way to do this to preserve backward compatibility.